### ISSUE TITLE
onMessageCompose not working on opening a Draft

### ISSUE THREAD

**Issue by Flyingdot** [OfficeDev/office-js#5173]
We are using the onMessageCompose-Event to display a message bar to the user in some situations. It works fine when using the preview in the reading pane. However, if I open the E-Mail in a new window from there with a double-click, the onMessageCompose-Event does not fire. This behaviour is only on Outlook for Mac; it is working fine in Windows (classic and new).

## Your Environment
* Platform [PC desktop, Mac, iOS, Office on the web]: Mac (new)
* Host [Excel, Word, PowerPoint, etc.]: Outlook
* Office version number: 16.91.1 (24111613)
* Operating System: macOS
 
## Expected behavior
the onMessageCompose-Event should fire if I open a draft in a new window (double-click).

## Current behavior
the onMessageCompose-Event does not fire.

## Steps to reproduce
1. Create an E-Mail and save it as a draft
2. Open the draft in the reading-pane -> onMessageCompose fires
3. Open the draft with a double-click in a new window -> onMessageCompose does not fire

## Context
This leads to inconsistent behavior.

**Comment by rajjha-msft** [comment]
Hey @Flyingdot 

We were not able to reproduce the issue. 
- Is this a consistent repro for you ?
- Does your add-in works fine on other platforms ?
- Can you please share the logs : https://github.com/OfficeDev/office-js/wiki/Collecting-log-files-for-Outlook-Add%E2%80%90ins-issues ?

**Comment by Flyingdot** [comment]
Yes it's a consistent repro. It only happens in the macOS client, all other platforms are ok.

Incident ID for the logs: 2RKSF6FW

**Comment by rajjha-msft** [comment]
Also, can you please share a repro video.

**Comment by Flyingdot** [comment]
https://github.com/user-attachments/assets/6c2a6058-4a2e-4ce2-b792-018d33950b51

**Comment by rajjha-msft** [comment]
Thanks for reporting this issue regarding OnMessageCompose launch event on Mac. It has been put on our backlog. We unfortunately have no timelines to share at this point 

Internal tracking id: Office: 5163064
